### DISCUSSION TITLE
Importance of SDSs, other than Network services?

### DISCUSSION THREAD

**Discussion by hallinpihlatie** [INSPIRE-MIF/helpdesk#125] [Q&A]
Hi,

Finland would like make an inquiry regarding the importance of Spatial Data Services (other than Network services) and of providing metadata for them. The last deadline of INSPIRE next year is related to this, and more precisely:
“10-Dec-2021 All invocable spatial data services shall be conformant with Annexes VI and (where practicable) VII of [INS ISDSS]”
But to be honest there is also a gap in meeting earlier deadlines related to these kinds of services..
 
As a background, is doesn’t seem to be much focus on them in the present EU level SDI as:
-	in the INSPIRE Geoportal focus is put, it seems, only on view and download services.
-	the web site which should provide examples of SDS from the yearly Member States’ monitoring is empty: http://inspire.ec.europa.eu/id/document/tg/sds/4.0/examples
-	they are not highlighted in the MIG work program 2020-2024
 
So Finland wonders, what the priority of SDSs (other than the view and download (network) services) actually is so that we can allocate resources on the coordination and implementation of this accordingly next year. Could you please give us some guidance on this?
 
Best regards,
Lena

**Comment by iuriemaxim** [comment]
@hallinpihlatie 

Although they are related to the INSPIRE validator too:

- this question and discussion related to the deadline for 2021 could be similar: https://github.com/INSPIRE-MIF/helpdesk-validator/issues/362

- this question related to the 2020 deadline could be relevant as well: https://github.com/INSPIRE-MIF/helpdesk-validator/issues/361

- this question related to various INSPIRE services could be relevant https://github.com/INSPIRE-MIF/helpdesk/issues/25

- this change request related to ATOM serving PNGs and PDF files could be also related https://github.com/INSPIRE-MIF/helpdesk-validator/issues/328

- this one if WPS are Network services or not https://github.com/INSPIRE-MIF/helpdesk/issues/26

- or even this one related to harmonized layers https://github.com/INSPIRE-MIF/helpdesk-validator/issues/39
